ui/pages/confirmations/components/security-provider-banner-alert/blockaid-banner-alert/blockaid-banner-alert.js

```diff
@@ -77,6 +110,35 @@ function BlockaidBannerAlert({ txData, ...props }) {
   } = securityAlertResponse;
 
   if (resultType === BlockaidResultType.Benign) {
+    if (isLoadingDisplayed) {
+      return (
+        <Box
+          alignItems={AlignItems.center}
+          borderColor={BorderColor.borderDefault}
```

Comment:
```suggestion
          borderColor={BorderColor.borderMuted}
```
to match design and origin border below it